### PR TITLE
Added dynamic properties.

### PR DESCRIPTION
**Pull request by romulogcerqueira**
* Added dynamic properties: range, limit_angle_left, limit_angle_right, step_angle, ping_pong_mode.
* Added resources folder with some models.
* Clear code.

### CODE REVIEW

File: gpu_sonar_simulation.orogen [hunk 1]
```diff
@@ -13,4 +13,17 @@ task_context "ScanningSonarTask" do
 	# Output data (Sonar beams)
    	output_port("beam_samples", "base::samples::SonarBeam");
    	periodic 0.1
+   	
+   	
+   	# Dynamic properties
+   	property("range", "double", 50.0).doc("Sonar Range").dynamic
```
Comment (doudou) [inline gpu_sonar_simulation.orogen]: Please document the properties with a comment block, NOT by calling .doc:

~~~
# Range of the sonar
property("range", "double", 0.5)
~~~

The common syntax is to leave the dot at the end and go to the next line:

~~~
property("range", "double", 50).
   dynamic
~~~

**Comment by doudou** [comment]
Please do NOT include the flatfish and manifold files in there. They are big, and are already included in the various flat fish packages.

**Comment by romulogcerqueira** [comment]
Fixed!

**Comment by doudou** [comment]
Git clones the whole history on every machine. Which means that if the file(s) have been there, they *will* be there forever.

So, for this particular case, you will have to really remove the files from whichever commit added it. Creating a new commit that removes them is not enough . Ask Gustavo and/or Thomio they can explain how to do that.

**Comment by romulogcerqueira** [comment]
Hi Sylvain,

I didn't know about this process.

Gustavo helped me to revert the 3 last commits to fix and remove the resources/ folder.

I am reopening this pull request.

**Comment by doudou** [comment]
> I didn't know about this process.

Not surprising, that's pretty advanced git

:+1:

**Pull request merged**